src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types';
+
+type TodoListProps = {
+  todos: Todo[];
+};
+
+export const TodoList = ({ todos }: TodoListProps) => (
```

Comment:
```suggestion
export const TodoList: React.FC<TodoListProps> = ({ todos }) => (
```

use proper typing for components